src/qml/QFieldCamera.qml

```diff
@@ -101,6 +104,27 @@ Popup {
 
         active: cameraItem.visible && cameraPermission.status === Qt.PermissionStatus.Granted
 
+        function applyCameraFormat() {
+          if (cameraSettings.pixelFormat != 0) {
+            let fallbackIndex = -1;
+            let i = 0;
+            for (let format of camera.cameraDevice.videoFormats) {
+              if (format.resolution == cameraSettings.resolution && format.pixelFormat == cameraSettings.pixelFormat) {
+                camera.cameraFormat = camera.cameraDevice.videoFormats[i];
```

Comment:
seems good to me :+1: 
is it possible to use `format` instead of `camera.cameraDevice.videoFormats[i]`

Reply:
@mohsenD98 , well spotted.